File: meeting_3/fooBarV0.cpp
#include <iostream>

using namespace std;

void fooBarV0(int num);

int main() {    
    fooBarV0(15);
    fooBarV0(42);
    fooBarV0(100);
}

void fooBarV0(int num) {
    cout << "Foo Bar Game of " << num << endl;
    for (int i = 1; i <= num; i++) {
        if (i % 15 == 0) {
            cout << "FooBar" << endl;
        } else if (i % 3 == 0) {
            cout << "Foo" << endl;
        } else if (i % 5 == 0) {
            cout << "Bar" << endl;
        } else {
            cout << i << endl;
        }
    }
}

File: meeting_3/fooBarV0.js
function fooBarV0(num) {
    console.log(`Foo Bar Game of ${num}`);
    for (let i = 1; i <= num; i++) {
        if (i % 15 == 0) {
            console.log('FooBar');
        } else if (i % 3 == 0) {
            console.log('Foo');
        } else if (i % 5 == 0) {
            console.log('Bar');
        } else {
            console.log(i);
        }
    }
}

fooBarV0(15);
fooBarV0(42);
fooBarV0(100);
